<?xml version="1.0" encoding="ISO-8859-15"?>

<launch>

    <node pkg="save_load_mesh" type="gl_viewer" name="gl_viewer" output="screen"
          args="/home/timer/temp/demo.ply /home/timer/bags/model_blur/zero_frame.txt"
    />

</launch>
